Utils/WorldZone.cs

```diff
@@ -0,0 +1,162 @@
+            Zones.Add(new WorldZone("PalFor", "Paleto Forest", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("PalHigh", "Palomino Highlands", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("Paleto", "Paleto Bay", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("Palmpow", "Palmer-T. Pow. Sta.", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("ProcoB", "Procopio Beach", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("Prol", "Ludendorff", EWorldZoneCounty.NorthYankton));
+            Zones.Add(new WorldZone("RANCHO", "Rancho", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("RGLEN", "Richman Glen", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("RTRAK", "Redwood Lights Track", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("Richm", "Richman", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("Rockf", "Rockford Hills", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("SKID", "Mission Row", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("STRAW", "Strawberry", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("SanChia", "San Chiansky Mountains", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("Sandy", "Sandy Shores", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("Slab", "Stab City", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("Stad", "Maze Bank Arena", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("TEXTI", "Textile City", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("Tatamo", "Tataviam Mountains", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("Termina", "Terminal", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("TongVaH", "Tongava Hills", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("TongvaV", "Tongava Valley", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("VCana", "Vespucci Canals", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("Vesp", "Vespucci", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("Vine", "Vinewood", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("WVine", "West Vinewood", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("WindF", "Ron Alt. Wind Farm", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("ZP_ORT", "Port of South LS", EWorldZoneCounty.LosSantos));
+            Zones.Add(new WorldZone("Zancudo", "Zancudo River", EWorldZoneCounty.BlaineCounty));
+            Zones.Add(new WorldZone("zQ_UAR", "Davis Quartz Quarry", EWorldZoneCounty.LosSantosCounty));
+            Zones.Add(new WorldZone("SanAnd", "San Andreas", EWorldZoneCounty.SanAndreas));
+        }

```

Comment:
Could all be done in the constructor saving one if-call every time calling GetZoneAtPosition as well as possible lag at first call due to List population.

Reply:
Never experienced any lag using this in singleplayer, even in demanding settings.

Reply:
Still does not seem to me as a good practice and I don't see any reason to do it like this.